pages/common/zcat.md

```diff
@@ -0,0 +1,11 @@
+# zcat
+
+> Print data from gzip compressed files.
+
+- Print the uncompressed contents of a gzipped file to the standard output:
+
+`zcat {{file.txt.gz}}`
+
+- Filter the contents of a compressed file with `grep`:
+
+`zcat {{file.txt.gz}} | grep {{pattern}}`
```

Comment:
Then there's no practical example of why zcat is useful: feeding a pipeline from a compress file.